earth_enterprise/src/third_party/postgis/SConscript

```diff
@@ -108,6 +108,8 @@ postgis_install = postgis_env.Command(
         'rm -rf %s/lib\n'
```

Comment:
Grumble: This formatting and `%s` expansion is a good way to prevent developers from reading these commands.

Can we you use a multi-line string and named variable expansion, instead?  E.g.:

```Python
        """
        cd {build_root}
        make DESTDIR={install_root} install
        rm -rf {install_root_opt}/lib
        mkdir -p {install_root_opt}/share/doc/packages/{ge_version}
        cp -pr COPYING CREDITS README.postgis {install_root_opt}/share/doc/packages/{ge_version}
        mkdir -p {install_root_opt}/share/postgresql/extension
        cp -pr {build_root}/liblwgeom/.libs/*.so* {install_root_opt}/share/postgresql/extension
        cd {install_root}{root_dir}
        tar cf - * | (cd {install_root_opt}; tar xf -)
        cd {install_root_opt}/share/postgresql/contrib
        mv {postgis_major_version}/* .
        rm -rf {postgis_major_version}
        cd {install_root}
        touch {postgis_target}
        """.format(**locals()))])
```

Reply:
Also a great idea - but beyond the scope of this PR, IMO.  This fixes a showstopper where GEE could not be installed or tested.